GOVERNANCE.md

```diff
@@ -280,6 +280,8 @@ All decisions affecting containerd, big and small, follow the same 3 steps:
 
 * Step 3: Merge or refuse the pull request. Who does this depends on the nature
 of the pull request and which areas of the project it affects.
+A pull request that contains potential breaking changes should be kept open for
+seven days at least.
```

Comment:
I don't object to this, but 7 days feels like a magic number.  Was there some specific reason that period was chosen?  If so, can we note it in this file?

Reply:
Other voting motions have seven days as well

Reply:
How do we classify a breaking change? Like if it breaks users who indirectly use containerd like k8s?

Reply:
Not specific to k8s though.

Reply:
I understand what we are trying to prevent and in support generally; in this case, if we (maintainers) are not aware of external community impacts (e.g. breaking for K8s, not for containerd directly), how will we know when to follow this? Given this is in response (I think?) to the RRO issue, were you aware of the breaking potential and could have marked the PR as such? I just want to make sure this isn't an additional policy that we aren't actually sure how to implement.

Reply:
For the RRO issue, I was aware of that it was a breaking change, but I wasn't thinking that it was breaking much

https://github.com/kubernetes/enhancements/pull/3858#discussion_r1351281024
```
Plan B is to keep the Kubernetes Core API and the CRI API completely unmodified,
and just let the CRI runtime treat "readonly" as "recursive readonly".

This would be much easier to implement and adopt, however, small portion of users may find this to be a breaking change.
```

Reply:
Updated as suggested. Sorry for the delay.